### ISSUE TITLE
Pagespeed Insights - Accessibility Score Is Unavailable

### ISSUE THREAD

**Issue by neighbor-peace** [GoogleChrome/lighthouse#15380]
### FAQ

- [X] Yes, my issue is not about [variability](https://github.com/GoogleChrome/lighthouse/blob/main/docs/variability.md) or [throttling](https://github.com/GoogleChrome/lighthouse/blob/main/docs/throttling.md).
- [X] Yes, my issue is not about a specific accessibility audit (file with [axe-core](https://github.com/dequelabs/axe-core) instead).

### URL

https://www.thrifty.com/us/en

### What happened?

The accessibility score is not provided when testing https://www.thrifty.com/us/en on PageSpeed Insights. The _Background and foreground colors have a sufficient contrast ratio_ and _Elements with visible text labels have matching accessible names_ audits are throwing the following error:

`axe-core Error: Failed to execute 'getImageData' on 'CanvasRenderingContext2D': Value is not of type 'long'.`

![image](https://github.com/GoogleChrome/lighthouse/assets/86714629/06ed6c15-3458-4b83-adda-d4cea4c7f86c)

Reproduce by testing https://www.thrifty.com/us/en on PageSpeed Insights

### What did you expect?

An accessibility score should be provided.

### What have you tried?

I've tried to isolate the problem by running Lighthouse against the page (10.0.0, 10.1.0, 10.2.0, 11.0.0) and by running axe-core tests on the page (4.6.3, 4.7.0, 4.7.1, 4.7.2). However, none of those tests throw any errors. This leads me to believe the issue is an edge case at the PageSpeed Insights layer caused by our environment.

The issue bears similarity to:
- https://github.com/dequelabs/axe-core/issues/3918
- https://github.com/GoogleChrome/lighthouse/issues/15002

However, those issues were reproducible by running Lighthouse and axe-core tests.

### How were you running Lighthouse?

PageSpeed Insights

### Lighthouse Version

10.0.0, 10.1.0, 10.2.0, 11.0.0

### Chrome Version

115.0.5790.170

### Node Version

16.16.0

### OS

Mac

**Comment by adamraine** [comment]
Fixing https://github.com/GoogleChrome/lighthouse/issues/15002 should also fix this issue, although I'm not sure why this would only fail in PageSpeed. It's possible that the content of the page is different in PageSpeed (the screenshots are different between PageSpeed and normal Lighthouse)

**Comment by neighbor-peace** [comment]
Thank you for the reply @adamraine.

For context, all traffic to that address is served the same content. It appears that the difference between screenshots is due to the snapshot being taken at different points during the rending of the page. Couldn't the difference be due to slight differences in the emulation environments? 

As for #15002, do you believe fixing it will fix this issue as well because the axe-core error will be treated as `score: 0` instead of `score: NaN`? Or because the underlying axe-core error will be resolved?

Also, on the off chance that you are able to, could you provide any tentative timeline for when this fix will be released?

**Comment by adamraine** [comment]
> For context, all traffic to that address is served the same content. It appears that the difference between screenshots is due to the snapshot being taken at different points during the rending of the page. Couldn't the difference be due to slight differences in the emulation environments?

It's possible. For reference this is what the screenshots look like on my end:

| CLI | PageSpeed Insights |
| --- | -------------------- |
| ![image](https://github.com/GoogleChrome/lighthouse/assets/6752989/d3d08c64-7eca-4d94-8c94-914f75b96b29) | ![image](https://github.com/GoogleChrome/lighthouse/assets/6752989/8029d676-7a8b-49d4-b7bb-e2d0de671e07) |

> As for https://github.com/GoogleChrome/lighthouse/issues/15002, do you believe fixing it will fix this issue as well because the axe-core error will be treated as score: 0 instead of score: NaN? Or because the underlying axe-core error will be resolved?

It will be because the underlying axe-core error is fixed.

> Also, on the off chance that you are able to, could you provide any tentative timeline for when this fix will be released?

It appears that a fix has landed in axe-core, however they haven't released it yet. We can't fix the changes on our end until the axe-core fix is released. Unfortunately, I don't have much information beyond that.

**Comment by adamraine** [comment]
The a11y issue is the same as https://github.com/GoogleChrome/lighthouse/issues/15002. Since the page appears to be serving different content for PSI and Lighthouse there isn't much we can do here.
